lean4-infoview/src/infoview/messages.tsx

```diff
@@ -109,7 +113,7 @@ export function AllMessages({uri: uri0}: { uri: DocumentUri }) {
                 </a>
             </span>
         </summary>
-        <MessagesAtFile uri={uri} messages={diags} />
+        <MessagesAtFile uri={uri} />
```

Comment:
This change breaks the pausing logic. Note that `diags` is frozen when the UI is paused, but with this change we ignore the frozen value. Since the pausable state is in this class rather than `MessagesAtFile`, one option may be to move `useMessagesForFile` up into this class and get rid of `MessagesAtFile`.

Reply:
This is ugly.. we don't want to move `useMessagesForFile` into this class because this will request all interactive diagnostics for the file (which blocks and causes a lot of unnecessary RPC messages).

Reply:
Okay, put in a moderately ugly solution using a thunk for the interactive diagnostics.